src/smspark/bootstrapper.py

```diff
@@ -233,15 +234,15 @@ def start_spark_standalone_primary(self) -> None:
         subprocess.Popen(cmd_start_primary, shell=True)
 
     def deserialize_user_configuration(
-        self, configuration_dict_or_list: Union[Dict[str, Any], List[Dict[str, Any]]]
+            self, configuration_dict_or_list: Union[Dict[str, Any], List[Dict[str, Any]]]
```

Comment:
nit: too much whitespace. Same with L245, L369

Reply:
this is consistent with argument indentation in other functions such as  line 376